### PR TITLE
Add MD80 product page 

### PR DESCRIPTION
**Pull request by mabrobotics**

### CODE REVIEW

File: _posts/2022-08-05-md80.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+- 'MD80'
+- 'QDD'
+- 'actuator'
+- 'brushless'
+- 'controller'
+- 'motor'
+- 'walking robot'
+twitter_text: Checkout the MD80
+website: https://www.mabrobotics.pl/servos
+wiki_homepage:
```
Comment (tfoote) [inline _posts/2022-08-05-md80.md]: This is a required field which should link to a page with the ROS developer documentation.

**Review by tfoote — CHANGES_REQUESTED** [review]
The icon should be 80x80: https://robots.ros.org/contributing/

Please make sure to make a wiki page with the developer documentation.

File: _posts/2022-08-05-md80.md (hunk 1 above)
Comment (mabrobotics) [reply _posts/2022-08-05-md80.md]: How can we set up a wiki page? We cannot find any create page button, apart from the create my homepage button in our profile (not sure if homepage should be about the product though)

**Review by tfoote — APPROVED** [review]
Thanks for filling in the wiki content.

**Pull request merged**